public_html/wp-content/themes/pattern-directory/single-wporg-pattern.php

```diff
@@ -38,16 +38,12 @@
 
 				<div
 					hidden
-					class="pattern-preview__container"
+					class="pattern__container"
```

Comment:
I coincidentally made the same change in:
https://github.com/WordPress/pattern-directory/pull/232/files#diff-a8a05ff78c3ae805334de62d6ed0efeb453b125ee358026d5dc43247ec90b13aR37